server/tests/common/replicated_loglet.rs

```diff
@@ -134,7 +134,7 @@ where
         cluster.wait_healthy(Duration::from_secs(30)).await?;
 
         let loglet_params = ReplicatedLogletParams {
-            loglet_id: ReplicatedLogletId::new(1),
+            loglet_id: ReplicatedLogletId::new(LogId::from(1u32), SegmentIndex::OLDEST),
```

Comment:
This need to be done also for the `reconfigure` subcommmand

Reply:
Thanks for the pointer. Will apply it.